harvard-university-of-worcester.csl

```diff
@@ -25,7 +25,7 @@
   </macro>
   <macro name="author">
     <names variable="author">
-      <name and="symbol" delimiter-precedes-et-al="never" delimiter-precedes-last="never" et-al-min="19" et-al-use-first="19" initialize-with="." name-as-sort-order="all"/>
+      
```

Comment:
I don't think you'll want to delete the whole line here, do you? This would give you
Adam Smith and Karl Marx (2001)
in the bibliography, where I'm assuming you'd want something like
Smith, Adam & Marx, Karl (2001).